src/platform/site-wide/announcements/components/HomepageRedesignModal.jsx

```diff
@@ -13,64 +13,69 @@ function HomepageRedesignModal({ dismiss, vaHomePreviewModal }) {
   const skipLinkElement = document.getElementsByClassName('show-on-focus')[0];
   ariaExcludeArray.push(skipLinkElement);
 
+  // Prevents modal from showing when redirected to homepage for auth
```

Comment:
I would consider wrapping this in an if-statement, checking window.location.search, which by default is an empty string, and thus falsy. We won't need to parse the search on each re-render.

Reply:
good idea, added that in.